### ISSUE TITLE
Compilation error in 2.0.0-beta9

### ISSUE THREAD

**Issue by maxmeyers** [twilio/cocoapod-specs#82]
Similar to #75, but instead of a warning I'm setting an error. Strangely, this only happens if I try to use `TVONotificationDelegate`.

```
<module-includes>:1:1: error: umbrella header for module 'TwilioVoice' does not include header 'TwilioVoiceClient.h' [-Werror,-Wincomplete-umbrella]
#import "Headers/TwilioVoice.h"
```

I deleted `TwilioVoiceClient.h` from the Headers directory and that fixed the compilation error. Is it possible it was left in there by accident?

**Comment by bchen-twilio** [comment]
Hi @maxmeyers 

Does this error message keep showing up after cleaning the build in Xcode?
Indeed the `TwilioVoiceClient.h` is no longer needed but got still got included. We'll have it removed in the next release.

Thanks
-bobie

**Comment by maxmeyers** [comment]
Yep, it's happening even after a "Clean build folder". For now we can commit the framework with the deleted header to our repo, but it would be nice to be able to use Cocoapods!

**Comment by bchen-twilio** [comment]
Ah, sorry for the inconvenience @maxmeyers.
I've actually done a new build job locally to exclude `TwilioVoiceClient.h` from the framework and the built SDK worked well in the testbed. We'll get this fix in and push the next release ASAP.

Thanks again for reporting this.
